microk8s-resources/wrappers/microk8s-helm.wrapper

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -eu
+
+export PATH="$SNAP/usr/sbin:$SNAP/usr/bin:$SNAP/sbin:$SNAP/bin:$PATH"
+export LD_LIBRARY_PATH="$SNAP/lib:$SNAP/usr/lib:$SNAP/lib/x86_64-linux-gnu:$SNAP/usr/lib/x86_64-linux-gnu"
+
+if [ ! -f "${SNAP_DATA}/bin/helm" ]; then
+  echo "Helm not available, try enabling it with 'microk8s.enable helm'" >&2
+  exit 0
+fi
+
+source $SNAP/actions/common/utils.sh
+"${SNAP_DATA}/bin/helm" "$@"
```

Comment:
I think we should specify the `--kubeconfig` argument here so that `microk8s.helm` will talk by default to microk8s. Similar to what we do in:  https://github.com/ubuntu/microk8s/blob/feature/insecure-port/microk8s-resources/wrappers/microk8s-linkerd.wrapper#L25